nomad/state/state_store.go

```diff
@@ -13,12 +13,13 @@ import (
 	"github.com/pkg/errors"
 
 	"github.com/hashicorp/nomad/helper"
+	"github.com/hashicorp/nomad/nomad/event"
```

Comment:
bulk of the changes are in state_store.go, commenting to force github to open the diff